### PR TITLE
Technical competence - select technical qualification

### PR DESCRIPTION
**Pull request by ajaxscape**
https://eaflood.atlassian.net/browse/WE-834

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -256,6 +256,13 @@ Constants.CompanyStatus = {
   NOT_ACTIVE: `isn't active`
 }
 
+Constants.TechnicalQualification = {
```
Comment (duncannixon) [inline src/constants.js]: I would prefer these constants to be grouped with the other Dynamics constants inside Constants.Dynamics (line 187 on)

File: src/controllers/technicalQualification.controller.js [hunk 2]
```diff
@@ -1,15 +1,63 @@
...
+const TechnicalQualificationValidator = require('../validators/technicalQualification.validator')
+const CookieService = require('../services/cookie.service')
+const Application = require('../models/application.model')
 
 module.exports = class TechnicalQualificationController extends BaseController {
-  async doGet (request, reply) {
-    const pageContext = this.createPageContext()
+  async doGet (request, reply, errors) {
+    const pageContext = this.createPageContext(errors, TechnicalQualificationValidator)
+    const {WAMITAB_QUALIFICATION, REGISTERED_ON_A_COURSE, DEEMED_COMPETENCE, ESA_EU_SKILLS} = Constants.TechnicalQualification
```
Comment (duncannixon) [inline src/controllers/technicalQualification.controller.js]: Prefer Constants.Dynamics.TechnicalQualification

File: src/constants.js (hunk 1 above)
Comment (ajaxscape) [reply src/constants.js]: Changed as requested

File: src/controllers/technicalQualification.controller.js (hunk 2 above)
Comment (ajaxscape) [reply src/controllers/technicalQualification.controller.js]: Changed as requested

**Comment by ajaxscape** [comment]
Good catch!  I've added html-hint to my box of tricks!

**Pull request merged**